docs/postgres.rst

```diff
@@ -6,6 +6,17 @@ Postgres version 9.4 or later is known to work.
 Set up database
 ===============
 
+Create a user ``synapse_user`` with::
+
+ su - postgres
```

Comment:
this is installation-specific. Not all distributions will have a unix user called `postgres`.